README.md

```diff
@@ -31,12 +31,17 @@ provided at the download links. You will need the Python 3.6 version.
 
 To install Devito, including examples, tests and tutorial notebooks, 
 follow these simple passes:
-```
+
+```sh
+# get the code
 git clone https://github.com/opesci/devito.git
 cd devito
-conda env create -f environment.yml
-source activate devito
-pip install -e .
+
+# run tests
+docker-compose run devito /tests
```

Comment:
this assumes the presence of docker

Reply:
Added a link to the Docker documentation.